generate.py

```diff
@@ -85,9 +85,12 @@ def create_seed(filename,
                 quantization_channels,
                 window_size=WINDOW):
     audio, _ = librosa.load(filename, sr=sample_rate, mono=True)
+    audio = audio_reader.trim_silence(audio)
 
     quantized = mu_law_encode(audio, quantization_channels)
-    cut_index = tf.size(quantized) + tf.constant(window_size) - tf.constant(1)
+    cut_index = tf.cond(tf.size(quantized) - tf.constant(window_size) < 0,
+            lambda: tf.constant(window_size),
```

Comment:
Shouldn't this be `lambda: tf.size(quantized)`, as we want to use the entire sample.
I think `window_size` would be outside of the sample in this case.